images/linux/scripts/tests/Common.Tests.ps1

```diff
@@ -0,0 +1,5 @@
+Describe "PATH check" {
+    It "PATH is not quoted" {
+        $PATH | Should -Not -Be '"*"'

```

Comment:
May be `$PATH | Should -Not -Match '"'`?

Reply:
I guess PATH can contain '"' but should not be in "", please, correct me if I'm wrong